lib/room.rb

```diff
@@ -28,18 +28,14 @@ def empty_until_string
   end
 
   def minutes_to_next_event
-    if events.empty?
+    if events.empty? # No upcoming events at all
```

Comment:
Much easier to read!

How are the values of "minutes to next event" and "minutes to end of event" being used currently?

I wanted to understand their goal to see if they could be merged into a `minutes_until_owner_changes` method that would detect the impending state change - from one event to another or between empty and busy states.

Reply:
The only place these are currently used where the difference is significant is in the room occupancy indicator in Leeds, where it will give a different signal for "this meeting ends in 5 minutes, and then the room is empty" vs "this meeting ends in 5 minutes, and there is another one right after it". It could potentially be swapped for `minutes_until_owner_changes` and then a boolean to indicate `room_empty_after_this_event`.